src/editors/containers/VideoGallery/index.jsx

```diff
@@ -45,6 +47,14 @@ export const VideoGallery = () => {
     uploadError: messages.uploadVideoError,
   };
 
+  const thumbnailFallback = (
+    <Image
+      thumbnail
+      className="px-6 py-4.5 h-100"
+      src={videoThumbnail}
+    />
```

Comment:
Why is image used instead of Paragon's `<Icon />` component with a Paragon icon?

Reply:
@KristinAoki The VideocamOutlined icon was added in Paragon v21.6.0 ([47e4379](https://github.com/openedx/paragon/commit/47e4379d1a65f0495e4cf015ed175efd117c7e9c)).

The latest release on https://www.npmjs.com/package/@edx/paragon is 21.5.6 and a notice says that future versions will be publised as https://www.npmjs.com/package/@openedx/paragon.

Installing @openedx/paragon and importing VideocamOutlined from it works, but is it a good idea?